plugins/remote-config/api/api.js

```diff
@@ -839,9 +910,12 @@ plugins.setConfigs("remote-config", {
 
         var collectionName = "remoteconfig_parameters" + appId;
 
+        //TODO:validations
```

Comment:
Is this related to one of the bug tickets?

Reply:
Yes, and kinda no, Its more of a reminder for circling back to this later